plugins/catalog-backend/src/permissions/rules/isEntityKind.ts

```diff
@@ -27,11 +27,14 @@ export const isEntityKind = createCatalogPermissionRule({
   name: 'IS_ENTITY_KIND',
   description: 'Allow entities with the specified kind',
   resourceType: RESOURCE_TYPE_CATALOG_ENTITY,
-  apply(resource: Entity, kinds: string[]) {
+  schema: z.object({
+    kinds: z.array(z.string()),
```

Comment:
I think the description is particularly helpful when params are more complex. In particular it gives us the opportunity to specify whether the condition requires _one of_ or _all of_ the values in the array (in this case of course it only makes sense for it to be one of the values, but still wouldn't hurt to specify.

Reply:
Description added!